mods/persistence/modules/atmospherics/components/injector.dm

```diff
@@ -0,0 +1,18 @@
+/obj/machinery/atmospherics/unary/outlet_injector/cabled
+	uncreated_component_parts = list(
+		/obj/item/stock_parts/power/terminal/buildable,
+		/obj/item/stock_parts/radio/receiver/buildable,
+		/obj/item/stock_parts/radio/transmitter/on_event/buildable,
+	)
+
+/obj/machinery/atmospherics/unary/outlet_injector/cabled/populate_parts(full_populate)
```

Comment:
There's a better way to do this using stock part presets, but it may require some of my upstream changes so it's fine as it is for now.